skl2onnx/__init__.py

```diff
@@ -15,9 +15,9 @@
 __model_version__ = 0
 
 
-from .convert import convert_sklearn
-from ._supported_operators import update_registered_converter
-from ._parse import update_registered_parser
+from .convert import convert_sklearn # noqa
```

Comment:
it looks a very normal import, why # noqa?

Reply:
This import is not directly getting used in this file, so flake8 throws a warning.